deploy/crds/saas.3scale.net_apicasts_crd.yaml

```
@@ -214,9 +247,42 @@ spec:
                     pullSecretName:
                       type: string
                       description: Quay pull secret for private repository
+                pdb:
+                  type: object
+                  properties:
+                    enabled:
+                      type: boolean
+                      description: Enable (true) or disable (false) PodDisruptionBudget
+                    maxUnavailable:
+                      type: string
+                      description: Maximum number of unavailable pods (number or percentage of pods)
+                    minAvailable:
+                      type: string
```

I would add some regex validation, free string for this value may lead to errors.

```suggestion
                      type: string
                      pattern: "^[0-9]+%?$"
```

---

Fixed on https://github.com/3scale/saas-operator/pull/34/commits/bbc39908f6299a5d3c4338009e9556a227365d6c